packages/shared/src/styles/base.css

```
@@ -166,6 +167,7 @@ body {
   --theme-background-bun: theme('colors.bun.20');
   --theme-background-onion: theme('colors.onion.20');
   --theme-background-pepper: theme('colors.pepper.20');
+  --theme-background-cabbage-opacity-24: theme('colors.cabbage.40')24;
```

Just tagging for visibility.
```suggestion
  --theme-background-cabbage-opacity-24: theme('colors.cabbage.40')3D;
```